### ISSUE TITLE
Stuck awaiting volume in macOS install

### ISSUE THREAD

**Issue by chrisportela** [NixOS/nix#5478]
**Describe the bug**
Trying to run multiuser install and install gets stuck at await_volume function.

**Steps To Reproduce**
1. Use install script in nix 2.4 manual for mutli-user install
2. Say yes to most prompts except "encrypt"

**Expected behavior**
Finishes step of launching `darwin-store` Launch Daemon

**`nix-env --version` output**

**Additional context**
I downloaded the tar file myself and edited the await_volume function and I was able to continue further

**Comment by chrisportela** [comment]
Update: I have manually downloaded the tar again, but using curl in a private folder. I ran in to issues with macOS not trusting any binaries because I ran them from the `~/Downloads` folder.

Running it again, but this time allowing it to encrypt the `/nix` volume has allowed it to work correctly.

**Comment by abathur** [comment]
A few Qs:
- Have you previously had (or tried to install) Nix on this system prior to the install that broke, or was this the first attempt at a fresh install? Did you already have a Nix Store volume before running the install? 
- Do you have any substantive vimrc/plugin files? If so, and if it wasn't the first run, this might be the same issue as #5468 (the reporter there got stuck at this point once during troubleshooting; but this was on a second attempt to run the installer).

**Comment by chrisportela** [comment]
I did have the previous version installed. It was a single user install. First I tried to install and see if it would notice the install and it didn't seem to.  The install worked but somethings were broken and I got the SSL issues documented in another issue.

Then I tried to remove Nix fully as much as I could. I ran the `rm` command given, but there were other files I think that were not deleted. The docs do not fully document all the places to remove things for this version. I also had to fix giving my Terminal Full Disk Access so it could completely clean out and delete the `/nix` directory/volume. I also needed to delete and unload the various services in LaunchDaemons.

After I was pretty sure I removed everything I was running into the issue I described of it hanging waiting for the volume. I ran the command it was looking for while installing it unencrypted.

At one point I accidentally made it encrypted. I deleted the volume in diskutil.app and removed it from the synthetic.conf and fstab files. But it would automatically make it encrypted without asking after that anyways. That said it finally finished installing and I was able to setup nix-darwin and home-manager again to get my configuration back. 

I use home-manager on nix-darwin so I do think there is non-trivial configuration for Vim, but I only enabled stuff home-manager offers.
